sql/moz-fx-data-shared-prod/search_derived/acer_cohort_v1/query.sql

```diff
@@ -0,0 +1,10 @@
+-- Query for search_derived.acer_cohort_v1
+-- Pulls all client_ids that were using an Acer distribution in late 2022
+-- Query saved for posterity; will not be re-run
```

Comment:
@scholtzan would it make more sense to use `init.sql` here?

Reply:
yes, init.sql would be good here